src/Symfony/Bridge/Twig/Command/DebugCommand.php

```diff
@@ -50,19 +52,24 @@ protected function configure()
     {
         $this
             ->setDefinition(array(
-                new InputArgument('filter', InputArgument::OPTIONAL, 'Show details for all entries matching this filter'),
+                new InputArgument('name', InputArgument::OPTIONAL, 'The template name'),
+                new InputOption('filter', null, InputOption::VALUE_REQUIRED, 'Show details for all entries matching this filter'),
```

Comment:
This needs to be documented in the CHANGELOG

Reply:
Updated CHANGELOG.